support-workers/src/typescript/services/config.ts

```diff
@@ -0,0 +1,48 @@
+import { GetParameterCommand, SSMClient } from '@aws-sdk/client-ssm';
+import type { z } from 'zod';
+import type { Stage } from '../model/stage';
+import { awsConfig } from '../util/awsConfig';
+
+export async function getConfig<I, O, T extends z.ZodType<O, z.ZodTypeDef, I>>(
+	stage: Stage,
+	configKeyName: string,
+	schema: T,
+): Promise<O> {
+	const ssmClient = new SSMClient(awsConfig);
+	const params = {
+		Name: `/${stage}/support/support-workers/${configKeyName}`,
+		WithDecryption: true,
+	};
+	const command = new GetParameterCommand(params);
+	const response = await ssmClient.send(command);
+	return schema.parse(JSON.parse(response.Parameter?.Value ?? ''));
```

Comment:
should we throw (or at least log) something if the config is missing rather than assuming empty?